### PR TITLE
update introduction to camunda ids

### PR DESCRIPTION
**Pull request by christinaausley**
Updating all "Introduction to Camunda" IDs to `introduction-to-camunda`.

Resolves https://github.com/camunda/camunda-platform-docs/issues/1111#

### CODE REVIEW

**Review by akeller — DISMISSED** [review]
Wow this looks like an annoying PR.

I'm not skilled enough write regex, so I hate to even suggest this... but it feels like there is a better way to do these redirects. 

@christinaausley did you test to confirm this corrects the incorrect behavior?

File: static/.htaccess [hunk 1]
```diff
@@ -10,6 +10,23 @@ Options -Indexes -MultiViews
...
+RewriteRule ^docs/1.2/guides/introduction-to-camunda-cloud/(.*)$ /docs/1.2/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/1.2/guides/introduction-to-camunda-cloud$ /docs/1.2/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/1.1/guides/introduction-to-camunda-cloud/(.*)$ /docs/1.1/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/1.1/guides/introduction-to-camunda-cloud$ /docs/1.1/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/1.0/guides/introduction-to-camunda-cloud/(.*)$ /docs/1.0/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/1.0/guides/introduction-to-camunda-cloud$ /docs/1.0/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/0.26/guides/introduction-to-camunda-cloud/(.*)$ /docs/0.26/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/0.26/guides/introduction-to-camunda-cloud$ /docs/0.26/guides/introduction-to-camunda/ [R=301,L]
+RewriteRule ^docs/0.25/guides/introduction-to-camunda-cloud/(.*)$ /docs/0.25/guides/introduction-to-camunda/$1 [R=301,L]
+RewriteRule ^docs/0.25/guides/introduction-to-camunda-cloud$ /docs/0.25/guides/introduction-to-camunda/ [R=301,L]
```
Comment (pepopowitz) [inline static/.htaccess:28]: Are these redirects needed? All of the docs changed in this PR had their _`id`_ changed, but none had their `slug` change. The `id` is the one used by docusaurus to know where to route links, but the `slug` is the one that users see in their browser.

**Review by pepopowitz — APPROVED** [review]
Nice, thank you! My feedback is non-blocking.

File: static/.htaccess (hunk 1 above)
Comment (christinaausley) [reply static/.htaccess:28]: You may be right here @pepopowitz -- I'll remove and see if it still build fine. 👍

**Comment by christinaausley** [comment]
> Wow this looks like an annoying PR.
> 
> I'm not skilled enough write regex, so I hate to even suggest this... but it feels like there is a better way to do these redirects.
> 
> @christinaausley did you test to confirm this corrects the incorrect behavior?

Locally, this does correct the behavior on build. I'll double check this in stage 👍

File: static/.htaccess (hunk 1 above)
Comment (christinaausley) [reply static/.htaccess]: Built clean. Thanks for the tip @pepopowitz ! @akeller In reference to [this comment](https://github.com/camunda/camunda-platform-docs/pull/1105#discussion_r940700209), Steve and I talked today and I'm going to go ahead and launch a PR to test and possibly drop any excess redirects with slugs like this. I might also go through and label the sections of redirects according to why they're there (C8 migration, "overview" naming changes, etc) as a sort of "htaccess clean up".

Comment (akeller) [reply static/.htaccess]: I _love_ this! Please add some comments in there!

**Review by akeller — APPROVED** [review]

**Pull request merged**